### PR TITLE
Adds onions

### PR DESCRIPTION
**Pull request by QualityVan**
:cl: QualityVan
add: The botany vending machine now has a stock of onion seeds
/:cl:

Adds onions and red onions, with sprites by WJohnson.
Onions are essentially unexceptional but can be sliced and microwaved to make onion rings. More recipes/mutations to come maybe???
Red onions contain a weak blinding agent which is mostly good for being annoying and which doesn't cause any lasting damage.

hydroponics
All the onion-y bits.

### CODE REVIEW

**Comment by Expletive** [comment]
Make them slice-able with a knife, so you have have proper sandwiches.

**Comment by Jordie0608** [comment]
What if onions mutate into ¡¿Español?! onions which make you speak spanish when eaten?

**Comment by Niknakflak** [comment]
Make players cry when they cut them. Force it. Literally hardcode that bitch

**Comment by WJohn** [comment]
Aw, I wanted the pink onions to contain fluoro acid. Oh well.

**Comment by QualityVan** [comment]
Gimme sprites for sliced onions and sure.

**Comment by optimumtact** [comment]
do they have layers like ogres?

**Comment by Anonmare** [comment]
Lizards are like onions.

They smell, make you cry, if you leave 'em out in the sun, they get all brown, an' start sproutin' little white hairs...

**Comment by Bawhoppen** [comment]
We need more simple things like this, not everything has to be ultimate OP meme items

**Comment by FantasticFwoosh** [comment]
When applied to the neck slot

> I held back adding more plants because not long ago i and maybe this thread would have been screamed at for botany-creep.

**Comment by TehZombehz** [comment]
Thank you. I can now make french onion dip.

**Comment by Anonmare** [comment]
@FantasticFwoosh Finally I can be an overly French Frenchie and Hon-Hon at all the rosbifs

**Comment by improvedname** [comment]
Next up garlic

**Comment by improvedname** [comment]
FUCK TEARJUICE GOT IN MY EYES CAPTAIN

**Comment by DoctoreMad** [comment]
Garlic is already in /vg/ just go fetch it

File: code/modules/hydroponics/grown/onion.dm [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/obj/item/seeds/onion
+	name = "pack of onion seeds"
+	desc = "These seeds grow into onions."
```
Comment (Tacolizard) [inline code/modules/hydroponics/grown/onion.dm]: ogres are like onions

**Review by Tacolizard — CHANGES_REQUESTED** [review]

**Comment by banthebantz** [comment]
they need layers and they need to make people cry

**Comment by rsanchezbalo** [comment]
regular_onion
purple_onion

Made some onions slices, im not the best at spriting tiny icons :D

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 2]
```diff
@@ -35,6 +35,14 @@
 		number += E.bang_protect
 	return number
 
+/mob/living/carbon/is_mouth_covered(head_only = 0, mask_only = 0)
+	if( (!mask_only && head && (head.flags_cover & HEADCOVERSMOUTH)) || (!head_only && wear_mask && (wear_mask.flags_cover & MASKCOVERSMOUTH)) )
+		return 1
```
Comment (lzimann) [inline code/modules/mob/living/carbon/carbon_defense.dm]: `TRUE`

**Comment by nicbn** [comment]
Onions + computer frame = syndicate deep web computer?

**Comment by Bawhoppen** [comment]
Why do we have to keep adding new crazy meme weapons to science/antags when we can add things like this?

**Comment by octareenroon91** [comment]
>cry from cutting onion

Throw transparent smoke clouds of tear juice maybe?

**Pull request merged**